packages/flutter_tools/lib/src/ios/xcodeproj.dart

```diff
@@ -93,6 +97,7 @@ class XcodeProjectInterpreter {
   final Usage _usage;
 
   static final RegExp _versionRegex = RegExp(r'Xcode ([0-9.]+)');
+  static final RegExp _buildRegex = RegExp(r'Build version ([a-zA-Z0-9]+)');
```

Comment:
@jmagman let me know if this regex makes sense. Trying to get any letter and number after "Build version"

Reply:
Makes sense. You could also do `\w+` which also matches an underscore but that's okay.

There's really no reason for a second regex though, you can combine these into one regex and get the second match.
```suggestion
  static final RegExp _versionRegex = RegExp(r'Xcode ([0-9.]+).*Build version (\w+)');
```